src/api/app/views/webui/users/tokens/new.html.haml

```diff
@@ -0,0 +1,84 @@
+- operation_options = Token::OPERATIONS.map { |a| "Token::#{a}".constantize.token_name }
+
+.card
+  .card-body
+    %h3.mb-3 Create Token
+
+    .row
+      .col-12.col-md-10.col-lg-8
+        = form_with(model: @token, method: :post, local: false) do |f|
+          .form-row.d-none#created-token
+            .col-12.col-md-10.col-lg-9
+              %fieldset.form-group
+                = f.label(:string_readonly, 'Your new OBS Personal Access Token Secret', class: 'col-form-label col-form-label-lg')
+                .input-group
+                  = f.text_field(:string_readonly, class: 'form-control', readonly: true)
+                  .input-group-append#copy-to-clipboard
+                    %span.input-group-text
+                      %i.fas.fa-clipboard
+              %hr
+
+          .form-row
+            .col
+              .form-group
+                = f.label(:operation, class: 'col-form-label col-form-label-lg mr-2')
+                = f.collection_radio_buttons(:operation, operation_options, :to_s, :to_s, class: 'custom-control-input', required: true) do |radio|
+                  .custom-control.custom-radio.custom-control-inline
+                    = radio.radio_button(class: 'custom-control-input')
+                    = radio.label(class: 'custom-control-label')
+
+          .form-row
+            .col
+              = render partial: 'webui/shared/autocomplete', locals: { html_id: 'token[project_name]', label: 'Project name', required: false,
+                                                                       data: { source: autocomplete_projects_path(local: true) } }
+            .col
+              = render partial: 'webui/shared/autocomplete', locals: { html_id: 'token[package_name]', label: 'Package name', required: false,
```

Comment:
This field should be disabled until the project field is filled.

Reply:
Done.